README.md

```diff
@@ -6,4 +6,4 @@ And here is a list of features:
 - the very first feature
 - second frature
 - third
-
+- fifth feature pink windows
```

Comment:
Lets make them freen

Reply:
Thanks for the intro to Github and Ruby. 
Please do not delete this branch. Hope for a fruitful collaboration in the future )